### PR TITLE
Feature/static jamjar overlay

### PR DESCRIPTION
**Pull request by drewbanin**
This is a biiiiig PR because I broke `jamjarPlayer.directive.js` into a bunch of different files. I tried to simplify everything, but LMK if you see anything that looks strange!

### CODE REVIEW

File: src/app/components/jamjarPlayer/jamjarPlayer.scss [hunk 1]
```diff
@@ -93,6 +93,11 @@ videogular div.jamjar-layer {
     //transition: all 1s ease-out;  
 }
 
+.jamjar-line-text {
+  line-height: 30px;
+
+}
+
```
Comment (jesseng11) [inline src/app/components/jamjarPlayer/jamjarPlayer.scss]: as discussed, we can take out "Now Playing" from the bars. Also, for the .jamjar-line class, can we change the height to 15px and add opacity to 0.5? this way the user can see the video behind better

File: src/app/components/jamjarPlayer/jamjar.js [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+JamJar.prototype.loadVideo = function(concert_id, video_id) {
+  self.videoService.getVideoById(video_id, function(err, video) {
+    if (err) {
+      return console.error(err);
+    }
+
+    self.nowPlaying = new Video(video, {}, self.$sce);
+
+    self.videos.push(self.nowPlaying);
+  });
```
Comment (jesseng11) [inline src/app/components/jamjarPlayer/jamjar.js]: This change for individual video player needs to be reflected on the html page too :)

Comment (drewbanin) [inline src/app/components/jamjarPlayer/jamjar.js]: fixed!

File: src/app/components/jamjarPlayer/jamjarPlayer.scss (hunk 1 above)
Comment (drewbanin) [inline src/app/components/jamjarPlayer/jamjarPlayer.scss]: fixed!

**Comment by jesseng11** [comment]
approved!!

**Pull request merged**